package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Compile LightScript to JavaScript.",
   "main": "index.js",
   "scripts": {
-    "test": "node node_modules/.bin/mocha test",
+    "test": "mocha test",
```

Comment:
When you run a command through `npm run` it automatically adds that to the path.